### PR TITLE
Bs event edit

### PR DESCRIPTION
**Pull request by bensenescu**

### CODE REVIEW

**Comment by bensenescu** [comment]
I can't get spotsAvailible to pre-populate, any help with that would be great.

File: src/views/CreateEventView.vue [hunk 1]
```diff
@@ -1,67 +1,47 @@
 <template>
     <form @submit.prevent="onSubmit">
     <div class="container">
-        <input v-validate="'required|max:80'" v-model="event.name"
+        <input v-validate="'required|max:80'" v-model="event.title"
```
Comment (jackblanc) [inline src/views/CreateEventView.vue:4]: this is sick

File: src/views/CreateEventView.vue [hunk 2]
```diff
@@ -90,27 +70,60 @@
...
     return {
       /*
-        name: String,
-        date: Date,
-        location: String,
-        startTime: String,
-        endTime: String,
-        description: String,
-        whatToBring: String,
+        "title": STRING,
```
Comment (jackblanc) [inline src/views/CreateEventView.vue:89]: is this just for reference?

File: src/views/CreateEventView.vue [hunk 3]
```diff
@@ -90,27 +70,60 @@
...
+      },
       error: '',
     };
   },
+  computed: {
+    ...mapGetters('events', {
+      getEventById: 'getEventById',
+    }),
+    isEditing() {
+      return parseInt(this.eventId, 10) > -1;
```
Comment (jackblanc) [inline src/views/CreateEventView.vue:111]: can u add a comment here

File: src/views/CreateEventView.vue [hunk 4]
```diff
@@ -90,27 +70,60 @@
...
+    },
+  },
+  watch: {
+    eventId: {
+      immediate: true,
+      async handler() {
+        if (this.isEditing) {
+          const eventCopy = await api.getEvent(parseInt(this.eventId, 10));
```
Comment (jackblanc) [inline src/views/CreateEventView.vue:119]: why parseInt again?

File: src/views/CreateEventView.vue (hunk 2 above)
Comment (bensenescu) [reply src/views/CreateEventView.vue:89]: yes

**Pull request merged**